src/analysis/souffle/taint.dl

```diff
@@ -0,0 +1,77 @@
+//-----------------------------------------------------------------------------
```

Comment:
Does this file need an #ifndef guard for inclusion?

Reply:
That is a good question. However, I am not sure if Souffle has such guards. I guess we will find out soon enough. 😄

Reply:
It does. [Here is an example](https://github.com/google-research/raksha/blob/cb01ac2ace4cf94148c0567a5a506d1321bc1845/datalog-prototyping/ifc_plus_auth_logic_prototype/labelChecking.dl)